target/linux/ramips/mt7621/base-files/etc/hotplug.d/ieee80211/10_fix_wifi_mac

```diff
@@ -25,9 +25,12 @@ case "$board" in
 		[ "$PHYNBR" = "1" ] && \
 			macaddr_add "$(mtd_get_mac_binary factory 0x4)" 1 > /sys${DEVPATH}/macaddress
 		;;
-	jcg,q20)
-		[ "$PHYNBR" = "1" ] && \
-			macaddr_setbit_la "$(mtd_get_mac_binary Factory 0x4)" > /sys${DEVPATH}/macaddress
+	jcg,q20|\
+	yuncore,ax820)
+		if [ "$PHYNBR" = "1" ]; then
+			base_mac=$(macaddr_add "$(mtd_get_mac_binary Factory 0x4)" 1)
```

Comment:
Take 0xe000 directly instead of incrementing 0x4 by 1.

Please also add a comment to the commit message about this (after the other MAC address stuff):

`To avoid a random MAC address for 5 GHz, we deliberately chose to set it based on LAN address instead.`

Reply:
BTW you are changing q20 here as well. This should not happen. Just add a separate block.